website/app/styles/pages/home.scss

```
@@ -1,36 +1,173 @@
+body.application.index {
+  background-color: var(--doc-color-black);
+}
 
 .doc-page-home {
-  grid-area: sidebar / sidebar / content / content;
+  grid-area: sidebar / sidebar / stage / stage;
 }
 
-.doc-page-home__hero {
-  padding-bottom: 80px;
-  background-color: #727374;
+// top-right decoration
+
+.doc-page-home::before {
+  position: absolute;
+  top: 0; // important: this is relative to the `<body>` itself (there are no parents with a non-static position)
+  right: 0;
+  left: 0;
+  height: calc(200px + var(--doc-page-header-height));
+  background-image: url("/assets/illustrations/home-abstract-small.jpg");
+  background-repeat: no-repeat;
+  background-position: top right;
+  background-size: auto 100%;
+  content: "";
+  pointer-events: none;
+
+  @include breakpoint.medium-and-above() {
+    height: 40vw;
+    max-height: 700px;
+    background-image: url("/assets/illustrations/home-abstract.jpg");
+    background-size: contain;
+  }
 }
 
+
+// content (container)
+
 .doc-page-home__content {
-  // looking at the current design, seems it's not like the normal <Doc::Page::Content @breakthrough={{true}}>
-  // so I have to create a custom thing for now (later when designs will be final I'll review everything)
-  grid-area: content / content / _extra/ _extra;
-  width: 100%; // needed (otherwise it collapses, because is in a grid)
-  max-width: calc(1216px + 2 * var(--doc-page-stage-gutter-medium));
+  position: relative;
+  z-index: 1;
   margin: 0 auto;
-  // TODO check with heather what she wants to do with the vertical padding (responsive)
-  padding: 16px var(--doc-page-stage-gutter-small);
+  // visual top padding is controlled by the "hero" element, so we set 0 here
+  padding: 0 var(--doc-page-stage-gutter-small) 64px;
+  color: var(--doc-color-white);
 
-  @include breakpoint.medium () {
-    padding: 32px var(--doc-page-stage-gutter-medium);
+  @include breakpoint.medium() {
+    padding: 0 var(--doc-page-stage-gutter-medium) 64px;
   }
 
-  @include breakpoint.large () {
-    padding: 64px var(--doc-page-stage-gutter-medium);
+  @include breakpoint.large() {
+    // 1216px by design
+    max-width: calc(1216px + 2 * var(--doc-page-stage-gutter-large));
+    padding: 0 var(--doc-page-stage-gutter-large) 128px;
   }
 }
 
+
+// hero
+
+.doc-page-home__hero {
+  @include breakpoint.small() {
+    --padding-top: 260px; // 200 of decoration on top + 60px of negative space
+    --padding-bottom: 100px;
+    --title-font-size: 2.625rem; // 42px
+    --tagline-font-size: 1rem; // 16px
+  }
+
+  @include breakpoint.medium() {
+    --padding-top: 120px; // decoration is below the content
+    --padding-bottom: 100px;
+    --title-font-size: 3.75rem; // 60px
+    --tagline-font-size: 1.25rem; // 20px
+  }
+
+  @include breakpoint.large() {
+    --padding-top: 300px; // decoration is below the content
```

After seeing this in the browser, the top padding feels too spacious. [hashicorp.com](https://www.hashicorp.com/) centers its content to the graphic, so this is as close to that as possible. 

```suggestion
    --padding-top: 140px; // decoration is below the content
```

---

fixed in https://github.com/hashicorp/design-system/pull/807/commits/b3b254f69abd6eb56076236bcc35f1acdb0f75e3